src/gs-plugin-apk/gs-plugin-apk.c

```diff
@@ -663,16 +690,35 @@ refine_apk_package (GsPlugin *plugin,
   return TRUE;
 }
 
-gboolean
-gs_plugin_refine (GsPlugin *plugin,
-                  GsAppList *apps,
-                  GsPluginRefineFlags flags,
-                  GCancellable *cancellable,
-                  GError **error)
+static gboolean
+gs_plugin_apk_refine_finish (GsPlugin *plugin,
+                             GAsyncResult *result,
+                             GError **error)
 {
+  return g_task_propagate_boolean (G_TASK (result), error);
+}
+
+static void
+gs_plugin_apk_refine_async (GsPlugin *plugin,
+                            GsAppList *list,
+                            GsPluginRefineFlags flags,
+                            GCancellable *cancellable,
+                            GAsyncReadyCallback callback,
+                            gpointer user_data)
+{
+  g_autoptr (GTask) task = NULL;
+  g_autoptr (GError) error = NULL;
+
+  task = g_task_new (plugin, cancellable, callback, user_data);
+  g_task_set_source_tag (task, gs_plugin_apk_refine_async);
+
+  if (flags == GS_PLUGIN_REFINE_FLAGS_NONE)
+    g_task_return_boolean (task, TRUE);
+  return;
```

Comment:
You need braces around this if, otherwise we won't refine :)

Reply:
Done! Can't trigger the CI though because of Github crypto stuff